pkg/fleet/installer/default_packages.go

```diff
@@ -42,6 +43,14 @@ var packageDependencies = map[string][]string{
 	"datadog-apm-library-python": {"datadog-apm-inject"},
 }
 
+var apmPackageDefaultVersions = map[string]string{
+	"datadog-apm-library-java":   "1",
+	"datadog-apm-library-ruby":   "1",
```

Comment:
I think it's 2 now for ruby to get the guardrail, cc @TonyCTHsu

Reply:
yes we want to do 2 now that we've made the OCI migration